veps/sig-compute/135-automatic-NUMA/automatic-numa.md

```diff
@@ -0,0 +1,119 @@
+# VEP #135: Automatic VM NUMA passthrough Enablement Based on Node Topology
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [ ] (R) Target version is explicitly mentioned and approved
+- [ ] (R) Graduation criteria filled
+
+## Overview
+
+This proposal proposes an automated mechanism to enable NUMA passthrough within a VirtualMachineInstance (VMI) when the VMI's requested resources (vCPUs or memory) have exceeded the capacity of a single physical NUMA node on the host. This feature will improve the performance of large VMs by reducing cross-NUMA latency and making the underlying topology transparent to the guest VM, without requiring manual user configuration. The behavior will be controllable via a KubeVirt configuration flag.
```

Comment:
Would the feature be simpler if the automatic enablement would be not node dependent?